app/src/main/java/agency/five/codebase/android/movieapp/ui/movieDetails/MovieDetailsScreen.kt

```diff
@@ -0,0 +1,163 @@
+package agency.five.codebase.android.movieapp.ui.movieDetails
+
+import agency.five.codebase.android.movieapp.mock.MoviesMock
+import agency.five.codebase.android.movieapp.ui.component.*
+import agency.five.codebase.android.movieapp.ui.movieDetails.mapper.MovieDetailsMapper
+import agency.five.codebase.android.movieapp.ui.movieDetails.mapper.MovieDetailsMapperImpl
+import agency.five.codebase.android.movieapp.ui.theme.*
+import android.widget.GridView
+import androidx.compose.foundation.background
+import androidx.compose.foundation.gestures.scrollable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.lazy.grid.GridCells
+import androidx.compose.foundation.lazy.grid.LazyVerticalGrid
+import androidx.compose.foundation.lazy.grid.items
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import coil.compose.AsyncImage
+import androidx.constraintlayout.compose.ConstraintLayout as ConstraintLayout
+
+private val movieDetailsMapper: MovieDetailsMapper = MovieDetailsMapperImpl()
+
+val movieDetailsViewState = movieDetailsMapper.toMovieDetailsViewState(MoviesMock.getMovieDetails())
+
+@Composable
+fun MovieDetailsRoute(
+
+) {
+    val movieDetailsViewState by remember { mutableStateOf(movieDetailsViewState) }
+
+    MovieDetailsScreen(
+        movieDetailsViewState,
+    )
+}
+
+@Composable
+fun MovieDetailsScreen(
+    movieDetailsViewState: MovieDetailsViewState,
+    spacing: Spacing = Spacing(),
+) {
+
+    val scrollState = rememberScrollState()
+
+    Column(Modifier.verticalScroll(scrollState)) {
+        ConstraintLayout {
```

Comment:
There is no need for use of `ConstraintLayout` here. You can arrange composables as required by using `Box` composable. However, it's okay to leave it now as it is. Feel free to try implementing same layout without `ConstraintLayout` for practice 😄